paccmann_chemistry/models/vae.py

```diff
@@ -229,51 +224,62 @@ def generate_from_latent(
             end_token must be discarded.
         """
         n_layers = self.n_layers
-        latent_z = latent_z.repeat(n_layers, 1, 1)
-        # TODO: Is this a common thing to do?
+        n_directions = self.n_directions
```

Comment:
Some things went wrong here during merging this PR into master @drugilsberg. This piece of code is now visible on [`master`](https://github.com/PaccMann/paccmann_chemistry/blob/a0dca129b24c72b3be1695558b60ffc28d75b06e/paccmann_chemistry/models/vae.py#L227) although I refactored it when working on the bidirectional GRU (see 23a560f)